File: app/src/main/java/com/iamagamedev/mybrandnewgame/gameObjects/worldObjects/Grass.java
package com.iamagamedev.mybrandnewgame.gameObjects.worldObjects;

import com.iamagamedev.mybrandnewgame.gameObjects.GameObject;

/**
 * Created by Михан on 24.05.2017.
 */

public class Grass extends GameObject {

    public Grass(float worldStartX, float worldStartY, char type){
        final float HEIGHT = 1;
        final float WIDTH = 1;

        setHeight(HEIGHT);
        setWidth(WIDTH);

        setType(type);

        setBitmapName("grass");
        setBadBitmapName("grass");

        setWorldLocation(worldStartX, worldStartY, -1);
        setRectHitBox();
    }
    @Override
    public void update(long fps) {

    }
}

File: app/src/main/java/com/iamagamedev/mybrandnewgame/gameObjects/worldObjects/Grass.kt
package com.iamagamedev.mybrandnewgame.gameObjects.worldObjects

import com.iamagamedev.mybrandnewgame.Constants.ObjectNames
import com.iamagamedev.mybrandnewgame.gameObjects.GameObject

/**
 * Created by Михан on 24.05.2017.
 */
class Grass(worldStartX: Float, worldStartY: Float, type: Char) : GameObject() {
    override fun update(fps: Long) {}

    init {
        var type = type
        val HEIGHT = 1f
        val WIDTH = 1f
        height = HEIGHT
        width = WIDTH
        this.type = type
        bitmapName = ObjectNames.GRASS
        badBitmapName = ObjectNames.GRASS
        setWorldLocation(worldStartX, worldStartY, -1)
        setRectHitBox()
    }
}